app/controllers/jobs_controller.rb

```diff
@@ -0,0 +1,68 @@
+class JobsController < ApplicationController
+  before_action :set_job, only: [:show, :edit, :update, :destroy]
+  before_action :set_company, only: [:new, :create, :show, :edit, :update, :destroy]
+
+  def index
+    if params.include?("sort")
+      @jobs = Job.all.order(:level_of_interest).reverse
```

Comment:
i suggest extracting this order query into a method on your `Job`. it belongs in the model because it's dealing with your data!